label_studio_ml/default_configs/Dockerfile

```diff
@@ -15,7 +15,9 @@ COPY supervisord.conf /etc/supervisor/conf.d/
 
 WORKDIR /app
 
+COPY /models/*.pt /app/data/models/
```

Comment:
@ntlex What if `/models` doesn't exist? Looks like it's a very user/system specific folder.

Reply:
You are right, I could remove this line since it is elaborated in the the `README.md` to ensure that the user either has to  volumize or copy the model dir in the docker application.

Reply:
maybe it's better to check existence of this folder?

Reply:
I think I misunderstood your initial comment about the user specific directory. I have corrected the directory's path to `./models/*.pt` on the latest commit. Let me know if this is what you had in mind.